adversarial-nn/adversarial-nn.json

```diff
@@ -0,0 +1,36 @@
+{
+    "key": "adversarialnn",
+    "label": "Adversarial Neural Network Attack",
+    "description": "Demonstration of adversarial attacks on a neural network trained on a dataset of handwritten digits",
+    "logo": "/ndslabs-badge.png",
+    "image": {
+        "name": "cheesehub/adversarial-nn"
```

Comment:
I've added an automated Docker build to the ANN repo:
https://github.com/cheese-hub/Adversarial-Neural-Network

Built and pushed [successfully](https://github.com/cheese-hub/Adversarial-Neural-Network/actions/runs/7037689215/job/19153005189) :+1: